### ISSUE TITLE
KitKat Confirmation for Old People

### ISSUE THREAD

**Issue by d4rken** [d4rken/sdmaid-public#207]
Originally reported by: **Bitbucket: [Trasd](http://bitbucket.org/Trasd), GitHub: [Trasd](http://github.com/Trasd)**

---

In a recent email, you requested I add this subject as an enhancement suggestion.  Here is the email text (to refresh your memory, if you're old like me):

**Me:** _I assume you don't confirm deletions?  Could you add code to validate deletions and warn people this feature will not work on KK 4.4+?  Chances are I'll forget and try again._

**Your response:** _In some cases I am. Could you create an enhacment ticket for this in the tracker? Not sure yet if this is something that I need change only for duplicates or if it's a global change that then affects all deletions._

I suggest the KK file manipulation validation be added globally.

As a reminder, here, again, is what happened: I copied my old backups (which, in turn, contained older and older backups going back over four tablets) to my new 64Gb SD Card. I knew I had duplicate files because it is easier to just dump all of an old card's data on a new card and worry about it later (bad practice, I know, but when a new tablet is involved, all discipline seems to go right out the window).  I decided to clean this chaotic mess up.

I spent a couple hours with SD Maid searching for and deleting duplicates.  Ohhhh, I was finally getting somewhere, or so I thought.  It was only after seeing the same file show up in the same directory that I realized there was a problem.  It was at this point I remembered that SD Maid didn't have access to any other directory on my external SD Card any more than any other app.

I am asking that you add some kind of validation check for manipulated files on KK and remind old people like me that on an unrooted device, SD Maid does not have access to other apps' directories (thanks Google, brilliant move!).

Plus, as I'll forget all this in a month, it is likely I will do it again!  LOL Again, smooth move Google.....

---
- Bitbucket: https://bitbucket.org/d4rken/sd-maid/issue/208
